lib/ansible/modules/packaging/os/pulp_repo.py

```diff
@@ -535,9 +537,9 @@ def main():
         add_export_distributor=dict(default=False, type='bool'),
         feed=dict(),
         generate_sqlite=dict(default=False, type='bool'),
-        ca_cert=dict(aliases=['importer_ssl_ca_cert']),
-        client_cert=dict(aliases=['importer_ssl_client_cert']),
-        client_key=dict(aliases=['importer_ssl_client_key']),
+        feed_ca_cert=dict(aliases=['importer_ssl_ca_cert', 'ca_cert']),
```

Comment:
This is not a good idea, since `ca_cert` is already a module option. (Due to `url_argument_spec()` above.)

Either do
```suggestion
        feed_ca_cert=dict(aliases=['importer_ssl_ca_cert'']),
```
and make sure that `feed_ca_cert` is set to `ca_cert`'s value if not set, or name the option `ca_cert` and add `feed_ca_cert` and `importer_ssl_ca_cert` as aliases. (Then there's no way to separate these options.)

Reply:
https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/urls.py#L1410

I must be being dumb, but I don't see it...

Reply:
@felixfontein I also don't see it being used by url_argument_spec()?

Reply:
Hmm, I was pretty sure it was in there. Apparently, I'm wrong. Sorry!